pkg/runs/submitter.go

```diff
@@ -529,10 +531,7 @@ func (submitter *Submitter) validateAndCorrectParams(
 	}
 
 	if err == nil {
-		if params.Requestor == "" {
-			// Requestor has not been set. Default it to the current user id.
-			params.Requestor, err = submitter.env.GetUserName()
-		}
+		params.Requestor, err = submitter.env.GetUserName()
```

Comment:
If the requestor will always be the current user, I don't think we'd need the requestor in the `RunsSubmitCmdParameters` struct anymore (which also means removing it from the available flags passed in from the user) - is this intentional?

Reply:
This part should be code that was already there, and would be changed in a new issue that we are currently working on.